client/main.html

```diff
@@ -21,15 +22,12 @@
   <meta name="theme-color" content="#ffffff">
 
   <!-- FontAwesome Icons  -->
-  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp" crossorigin="anonymous" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"
```

Comment:
We could maybe include this pre-connect here for a better performance and web-vitals score

```suggestion
  <link rel="preconnect" href="https://fonts.googleapis.com/" crossOrigin="anonymous" />
  <link rel="dns-prefetch" href="https://fonts.googleapis.com/" />
  <link rel="preconnect" href="https://use.fontawesome.com/" crossOrigin="anonymous" />
  <link rel="dns-prefetch" href="https://use.fontawesome.com/" />
  
  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"
```

Reply:
^ this suggestion should move the `<!-- FontAwesome Icons -->` line too